src/main/java/Parser.java

```diff
@@ -0,0 +1,49 @@
+public class Parser {
+    public Parser() {
+    }
+
+    public ChatbotKeyword parse(String input) {
+        String[] inputArray = input.split(" ");
+        String command = inputArray[0].toLowerCase();
+        switch (command) {
+            case "list":
+                return ChatbotKeyword.LIST;
+            case "done":
+                return ChatbotKeyword.MARK;
+            case "delete":
+                return ChatbotKeyword.DELETE;
+            case "todo":
+                return ChatbotKeyword.TODO;
+            case "deadline":
+                return ChatbotKeyword.DEADLINE;
+            case "event":
+                return ChatbotKeyword.EVENT;
+            case "bye":
+                return ChatbotKeyword.BYE;
+            default:
+                throw new IllegalArgumentException("I'm sorry, but I don't know what that means :-(");
+        }
+    }
+
+    public String[] deadlineParse(String input) {
+        String[] deadlineArray = input.split(" /by ");
+        String deadlineDescription = deadlineArray[0];
+
+        return new String[]{deadlineDescription, deadlineArray[1]};
+    }
+
+    public String[] eventParse(String input) {
```

Comment:
Perhaps it is better to phrase the method as a verb, like `parseEvent`?

Reply:
thanks for raising this up @teojunda , I think its definitely a valid point that i missed